opentuner/search/manipulator.py

```diff
@@ -222,7 +223,7 @@ class Parameter(object):
   """
   __metaclass__ = abc.ABCMeta
 
-  def __init__(self, name):
+  def __init__(self, name, **kwargs):
```

Comment:
Normally you do that pattern to allow args to be added to your base class without changing the subclass.  You could pass kwargs up to the parent class here (object), which would throw an error for unknown args or you could remove it here.  Having classes complain about unknown args is good behavior as it helps for debugging.